content/docs/pipelines/sdk/install-sdk.md

```diff
@@ -71,7 +71,8 @@ up Python using [Miniconda](https://conda.io/miniconda.html):
 Run the following command to install the Kubeflow Pipelines SDK:
 
 ```bash
-pip install https://storage.googleapis.com/ml-pipeline/release/{{% pipelines-sdk-version %}}/kfp.tar.gz --upgrade
+latest_version=$(curl --silent https://api.github.com/repos/kubeflow/pipelines/releases/latest | jq -r .tag_name)
+pip install https://storage.googleapis.com/ml-pipeline/release/$latest_version/kfp.tar.gz --upgrade
```

Comment:
I think the variable should be enclosed in curly brackets:

```
pip install https://storage.googleapis.com/ml-pipeline/release/${latest_version}/kfp.tar.gz --upgrade
```

Reply:
updated. 

One question. Where is the code where it performs one deployment per branch ? Is it Jenkins ?

Reply:
It's Netlify that builds the doc deployment. To find the preview for your PR:

* Wait for the automated PR workflow to do some checks. When it's ready,
  you should see a comment like this: **deploy/netlify — Deploy preview ready!**
* Click **Details** to the right of "Deploy preview ready" to see a preview
  of your updates.